lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -170,6 +165,13 @@ mpl.figure.prototype._init_canvas = function() {
     // Set the figure to an initial 600x600px, this will subsequently be updated
     // upon first draw.
     this._resize_canvas(600, 600);
+
+    function set_focus () {
+        canvas.focus();
```

Comment:
Does the canvas tag have `tabindex` set or are `<canvas>` elements automatically fucusable?